CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 0.14.0 (05.09.2023)
+
+- Allow to add fixed fee when invoking functions.
```

Comment:
IMO this better suits `- Allow to add custom extra fee rate when invoking functions.`